state-chain/pallets/cf-pools/src/tests.rs

```diff
@@ -464,9 +465,25 @@ fn cant_schedule_in_the_past() {
 				0,
 				Some(0),
 				IncreaseOrDecrease::Decrease(55),
-				Some(9) // 1 block in the past
+				// 1 block in the past
+				Some(CURRENT_BLOCK - 1)
 			),
-			Error::<Test>::LimitOrderUpdateExpired
+			Error::<Test>::InvalidDispatchAt
+		);
+
+		assert_noop!(
```

Comment:
Would be nice to test that for `CURRENT_BLOCK + SCHEDULE_UPDATE_LIMIT_BLOCKS` this does work.